src/PKSim.Core/Model/SimulationFactory.cs

```diff
@@ -208,12 +208,13 @@ public Simulation CreateForVSS(Protocol protocol, Individual individual, Compoun
       public IndividualSimulation CreateForDDIRatio(IndividualSimulation originalSimulation)
       {
          //DDI Ratio=> manipulate inhibition parameters so that all DDI effects are effectively deactivated 
-         var ddiRatioSimulation = createModelLessSimulationBasedOn(originalSimulation);
+         var ddiRatioSimulation =  createModelLessSimulationBasedOn(originalSimulation);
 
          _simulationModelCreator.CreateModelFor(ddiRatioSimulation);
 
-         //now update all parameters from the orginal simulation
-         _simulationParametersUpdater.ReconciliateSimulationParametersBetween(originalSimulation, ddiRatioSimulation);
+         // now update all parameters from the original simulation. We need to take protocol parameters as well as simulation parameters into account for the update
+         // as simulation building block are all kept in sync with the changes values EXCEPT Protocol (1 to n mapping)
+         _simulationParametersUpdater.ReconciliateSimulationParametersBetween(originalSimulation, ddiRatioSimulation, PKSimBuildingBlockType.Simulation | PKSimBuildingBlockType.Protocol) ;
```

Comment:
The real change is here in that case. We need to update Simulation and Protocol parameters